maidsafe/safe-network-app
# Add Error/Warn/Info banners

resolves #15 

This PR is to add Notifications to the SAFE LAUNCHPAD

QA to test this 
1) Add an env variable before launching the `safe-network-app`
2) And an env variable for `NOTIFICATION` using `NOTIFICATION=NO_INTERNET`
3) Now launch the `safe-network-app` and check if a notification has been added to the Application

The different types of notifications that can be added are
- DISC_FULL
- NO_INTERNET
- GLOBAL_FAILURE
- ADMIN_PASS_REQ
- SERVER_TIMED_OUT
- UNINSTALL_APP_ALERT
- CLOSE_APP
- CLOSE_APP_ALERT
- UPDATE_AVAILABLE
- UPDATE_AVAILABLE_ALERT
- RESTART_SYSTEM
- RESTART_SYSTEM_ALERT
- CLEARNET_WARNING_ALERT

For reference as to how each of the notifications should look like - https://www.figma.com/file/53iCtsVlxk9S0N9fAGJ5nH/Launchpad-App-Material

By setting the environment variable of  `NOTIFICATION` to any of the above, a notification will be added to the application upon launching the application
